python/ray/train/torch.py

```diff
@@ -177,9 +177,15 @@ def __init__(self, base_dataloader: DataLoader, device: torch.device):
         self._dataloader = base_dataloader
         self._device = device
 
+    def _move_to_device(self, item):
+        return (i.to(self.device) for i in item)
```

Comment:
Do we need to validate the type of `item`? Is it safe to assume the `item` is always a `Tensor`?

Reply:
Ok updated to do a no-op if them item does not have a `.to` method.